privacyidea/lib/tokens/hotptoken.py

```diff
@@ -298,6 +298,11 @@ def get_init_detail(self, params=None, user=None):
                                                   "value": oath_url,
                                                   "img": create_img(oath_url)
                                                   }
+                except KeyError as ex:
+                    audit_entry = self.find_most_recent_audit_entry(action="POST /auth")
```

Comment:
I think you are mixing s.th. up?
The find_most_recent_audit_entry is a member function of the tests.

Please wirte a test, to cover this Exception.